### ISSUE TITLE
Mobile: Highlight grid rows on tap

### ISSUE THREAD

**Issue by TomTirapani** [xh/hoist-react#2476]
Investigate what support AgGrid has for highlighting a row on tap. Note this is different than selection, it should be transient feedback for the user.

**Comment by lbwexler** [comment]
Waiting on ag-Grid, as discussed in Pull Request.

**Comment by amcclain** [comment]
Should be unblocked when #2684 resolved

**Comment by TomTirapani** [comment]
New PR using AgGrid v26: https://github.com/xh/hoist-react/pull/2764
